silnlp/__init__.py

```diff
@@ -7,3 +8,5 @@
 formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
 ch.setFormatter(formatter)
 LOGGER.addHandler(ch)
+
+os.system("nvidia-smi")
```

Comment:
Much of silnlp does not require/use a GPU. It might be better to only call this on scripts that require a GPU.

Reply:
How about at the beginning of train?  The issue is that we wouldn't know at the beginning, but I guess that is ok in 95% of the cases.  Will change.